autoload/ale/handlers.vim

```diff
@@ -4,6 +4,47 @@ scriptencoding utf-8
 "   linter which outputs warnings and errors in a format accepted by one of
 "   these functions can simply use one of these pre-defined error handlers.
 
+function! s:HandleUnixFormat(buffer, lines, type)
+    " Matches patterns line the following:
+    "
+    " file.go:27: missing argument for Printf("%s"): format reads arg 2, have only 1 args
+    " file.go:53:10: if block ends with a return statement, so drop this else and outdent its block (move short variable declaration to its own line if necessary)
+    " file.go:5:2: expected declaration, found 'STRING' "log"
+    let l:pattern = '^.*:\(\d\+\):\(\d\+\): \(.\+\)$'
```

Comment:
There is one bug here. The column can be optional here. You'll have to use `?` in your regex to handle that.